tests/Bot.Builder.Community.Adapters.Tests/ApplicationBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Moq;
+using Xunit;
+using Microsoft.Extensions.Options;
```

Comment:
Please, re-order these alphabetically. It's a linter rule that is not mandatory in this repo yet, but we could get ahead.